OTKit/otkit-web-typography/token.yml

```diff
@@ -0,0 +1,74 @@
+props:
+  otkit-web-heading-xlarge-font-size:
```

Comment:
can't we just call this `heading-xlarge-font-size` - we are already inside `otkit-web`...

Reply:
I'm going back and forth on this level of verboseness (?) myself, but I'm ok doing it either way.

Reply:
I vote for verbose but not duplicate. Just imagine writing `@value otkit-web-heading-xlarge-font-size from 'otkit-web-typography/tokens.cssmodule.css`

Reply:
I changed the token name itself to `otkit-desktop-typography` but removed all the prefixes for the values. Can you re-review